main.go

```diff
@@ -33,6 +32,8 @@ func createManifest(ext string) (manifestInterface, error) {
 		return &manifestJSON{}, nil
 	case "yaml":
```

Comment:
Why not just `case "yaml", "yml":`

Reply:
I did not know that was a thing, I'll change it.